src/pranitaPJun20/UnionOfArraysUsingMethod.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Assignment 23 : 2nd July 2020
+
+Program 1: Find union of two arrays.
+Input: arr1 -> {10.45, 14.0, 18.35, 88.88, 54.10, 18.35}
+          arr2 -> {17.20, 13.30, 10.45, 18.35, 84.33, 13.30}
+          
+          Output: 10.45, 14.0, 18.35, 88.88, 54.10, 17.20, 13.30, 84.33
+Method Signature : double[] getUnionOfArray(double[] arr1, double[] arr2)
+ */
+package pranitaPJun20;
+
+public class UnionOfArraysUsingMethod {
+	int countUnionArray = 0;
+
+	void getUnionOfArrays(double[] array1, double[] array2) {
+		double[] tempUnionArray = new double[(array1.length + array2.length)];
+		boolean flag;
+		for (int outerIndex = 0; outerIndex < array1.length; outerIndex++) {
+			flag = true;
+			for (int index = 0; index < tempUnionArray.length; index++) {
```

Comment:
more optimal condition should be ,

index < countUnionArray

Reply:
Got it